### ISSUE TITLE
LSTM Cell example

### ISSUE THREAD

**Issue by CarlosJose126** [pytorch/pytorch#51801]
## 📚 Documentation

Hi, I think that there is an error in the example presented for a LSTMCell:
https://pytorch.org/docs/stable/generated/torch.nn.LSTMCell.html

I suggest using the same example given for the GRUCell: https://pytorch.org/docs/stable/generated/torch.nn.GRUCell.html
but adapted to the LSTM gates:

import torch.nn as nn
import torch

# features in the input: 10
# hidden size: 20
rnn = nn.LSTMCell(10, 20)
# (time_steps, batch, input_size)
input = torch.randn(2, 3, 10)
# (batch, hidden_size)
hx = torch.randn(3, 20)
# (batch, hidden_size)
cx = torch.randn(3, 20)
output = []
for i in range(input.size()[0]):
        hx, cx = rnn(input[i], (hx, cx))
        output.append(hx)       
        
output  = torch.stack(output, dim = 0)

**Comment by jbschlosser** [comment]
Hey @CarlosJose126, thanks for the great suggestion! Would you be willing to open a PR with the fix? Docs for `LSTMCell` are in `torch/nn/modules/rnn.py`.

**Comment by CarlosJose126** [comment]
Hi @jbschlosser, I just submit the pull request Example LSTMCell #51983. 
Thank you for your answer!

**Comment by zou3519** [comment]
PR has been merged and closed. Thank you @CarlosJose126 for the fix!
